lib/src/device_info/device_info.dart

```diff
@@ -35,7 +35,7 @@ class IosDeviceInfoImpl implements DeviceInfo {
   String get os => '${iosDeviceInfo.systemName} ${iosDeviceInfo.systemVersion}';
 
   @override
-  int? get sdk => 0;
+  int? get sdk => int.parse(iosDeviceInfo.systemVersion);
```

Comment:
Add tryParse instead of parse and provide `?? 0` at end

Reply:
Yes. System version cannot be null.